crates/swc_common/Cargo.toml

```diff
@@ -39,11 +39,12 @@ shrink-to-fit = ["dep:shrink-to-fit", "swc_atoms/shrink-to-fit"]
 
 
 [dependencies]
-anyhow                = { workspace = true, optional = true }
+anyhow                = { workspace = true }
 arbitrary             = { workspace = true, features = ["derive"], optional = true }
 bytecheck             = { workspace = true, optional = true }
 cfg-if                = { workspace = true }
 either                = { workspace = true }
+miette                = { workspace = true, features = ["fancy-no-syscall"] }
```

Comment:
It should not depend on `miette` directly. It's too heavy to be a common dependency

Reply:
I can move it to `swc_error_reports` crate

Reply:
Thank you! Also, I think `fancy-no-syscall` should be optionally enabled (using a cargo feature) because binary size matters for some users of SWC